include/MeshKernel/Contacts.hpp

```diff
@@ -61,8 +61,8 @@ namespace meshkernel
         /// The figure below shows two 2d meshes, a 1d mesh between them, and the 1d-2d contacts (in red).
         /// The boundary nodes of the 1d mesh (those sharing only one 1d edge) are not connected to any 2d face.
         /// For the 1d nodes not overlapping a 2d mesh,
-        /// a ray starting from the current node n is computed (dashed blue ray).
-        /// This ray is normal to the segment connecting the previous (n-1) and next one 1d node (n+1),
+        /// a ray starting from the current node m_n is computed (dashed blue ray).
```

Comment:
this change could be unintentional

Reply:
yes, indeed. i used replacement, but ReSharper was not activated. a lesson I just learned